package.json

```diff
@@ -33,35 +33,32 @@
   "license": "MIT",
   "scripts": {
     "build:node": "esbuild src/node.ts --outfile=lib/node.cjs --bundle --sourcemap --platform=node --external:cosmokit --external:supports-color --target=es2019",
-    "build:browser": "esbuild src/browser.ts --outfile=lib/browser.mjs --bundle --sourcemap --platform=neutral --external:cosmokit --external:util --target=esnext",
+    "build:browser": "esbuild src/browser.ts --outfile=lib/browser.mjs --bundle --sourcemap --platform=neutral --external:cosmokit --target=esnext",
     "build:deno": "esbuild src/deno.ts --outfile=lib/deno.mjs --bundle --sourcemap --platform=neutral --external:cosmokit --target=esnext",
     "build": "yarn build:node && yarn build:browser && yarn build:deno && yarn tsc -b",
     "test": "mocha -r esbuild-register tests/*.spec.ts",
     "test:text": "shx rm -rf coverage && c8 -r text yarn test",
     "test:json": "shx rm -rf coverage && c8 -r json yarn test",
-    "test:html": "shx rm -rf coverage && c8 -r html yarn test",
-    "dep": "yakumo upgrade"
+    "test:html": "shx rm -rf coverage && c8 -r html yarn test"
   },
   "devDependencies": {
-    "@sinonjs/fake-timers": "^6.0.1",
+    "@sinonjs/fake-timers": "^10.0.0",
```

Comment:
Why change major? This would break tests.

I really can't understand that you can achieve your needs simply, but you insist on introducing a bunch of irrelevant destructive changes on your own. Merging your PR without careful review will make this package unusable for everyone except you (probably including you).

This is an underlying library and I hope you understand what you are doing.

Reply:
I refuse to review further changes to this PR.

Please open another one and make sure you MAKE MINIMAL CHANGES.

Reply:
I have tested this and it does not affect the existing behavior.

Reply:
> I refuse to review further changes to this PR.
> 
> Please open another one and make sure you MAKE MINIMAL CHANGES.

ok

Reply:
> I have tested this and it does not affect the existing behavior.

You certainly haven't considered all situations, such as the browser behavior I've mentioned earlier.

Reply:
Makes sense.